tests/foreman/maintain/test_maintenance_mode.py

```diff
@@ -18,34 +18,39 @@
 from robottelo.config import robottelo_tmp_dir
 
 
+@pytest.mark.include_satellite_iop
 @pytest.mark.e2e
 def test_positive_maintenance_mode(request, sat_maintain, setup_sync_plan):
     """Test satellite-maintain maintenance-mode subcommand
 
     :id: 51d76219-d3cc-43c0-9894-7bcb75c163c3
 
     :setup:
-        1. Create a sync-plans which is enabled
-        2. set ':manage_crond: true' in /etc/foreman-maintain/foreman_maintain.yml
+        1. Get a Satellite(default and/or IoP-enabled Satellite)
+        2. Create a sync-plan which is enabled
+        3. set ':manage_crond: true' in /etc/foreman-maintain/foreman_maintain.yml
```

Comment:
I've just realized this is not needed since https://github.com/theforeman/foreman_maintain/commit/ccd089ca44d1be2442e3348fb036853680c2877e